.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,23 +4,18 @@ about: Create a report to help us improve
 
 ---
 
-#### Description
+### Description
```

Comment:
I'm into the formatting as H3. Saves space and the least important part (the subject line) is not as in your face. Can we have them as ###?

Reply:
I went with H2 for top level because the PR template has three tiers of titles as seen here: 

```
## Any other comments?
This is a great place to ask for specific feedback from people.

### Where has this been tested?
**Platforms (Desktop, Phone, Tablet,etc...** …

**Browsers** …
```
Since H4 and bold show as the same thing being able to differentiate the levels was important hence working backwards from the smallest up.